src/workerd/api/wpt/generate-tests.bzl

```
@@ -0,0 +1,86 @@
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("//:build/wd_test.bzl", "wd_test")
+
+# 0 - name of the service
+# 1 - es module file path (ex: url-test for url-test.js)
+# 2 - external modules required for the test suite to succeed
+WPT_TEST_TEMPLATE = """
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "{}",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "{}.js"),
+          (name = "harness", esModule = embed "../../../../../workerd/src/wpt/harness.js"),
+          {}
+        ],
+        bindings = [
+          (name = "wpt", service = "wpt"),
+        ],
+        compatibilityDate = "2024-07-01",
+        compatibilityFlags = ["nodejs_compat_v2"],
```

```suggestion
        compatibilityFlags = ["nodejs_compat_v2", "experimental"],
```

Should include experimental just to be safe.

---

Also, for compatibilityDate... could that be set dynamically to the current day whenever these are run? I don't want us to have to go back and manually change that repeatedly over time.

---

I've updated the implementation to use dynamic dates, and added experimental flag.